types/react-native/index.d.ts

```diff
@@ -5,6 +5,7 @@
 //                 HuHuanming <https://github.com/huhuanming>
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
+//                 Kamal Mahyuddin <https://github.com/kamal>
```

Comment:
Just wanting to make sure that you understand that this means you’ll be signing up as a maintainer and will get notifications for all issues/PRs.

Same goes to you @timwangdev.